src/vscode-bicep/src/extension.ts

```diff
@@ -7,6 +7,7 @@ import {
 } from "vscode-azureextensionui";
 
 import { launchLanguageServiceWithProgressReport } from "./language";
+import { BicepCacheContentProvider } from "./language/bicepCacheContentProvider";
```

Comment:
[nit] This can be merged with the import statement above (`./language` contains `index.ts` which exports everything in that folder).